solidity/script/avs/DeployNetwork.s.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+import "forge-std/Script.sol";
+
+import {ProxyAdmin} from "../../contracts/upgrade/ProxyAdmin.sol";
+import {TransparentUpgradeableProxy} from "../../contracts/upgrade/TransparentUpgradeableProxy.sol";
+import {TimelockControllerUpgradeable} from "@openzeppelin/contracts-upgradeable/governance/TimelockControllerUpgradeable.sol";
+
+contract TimelockControllerInitializer is TimelockControllerUpgradeable {
```

Comment:
Imo an odd name for a contract, maybe just call it Network or HyperlaneOmnibusNetwork or something like that??